packages/react/src/VideoPlayer/VideoPlayer.features.stories.tsx

```diff
@@ -58,3 +62,44 @@ export const Minimal = () => (
     <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
   </VideoPlayer>
 )
+
+const MyVideoPlayer = () => {
+  const {isPlaying, togglePlaying, seek} = useVideo()
+
+  return (
+    <Stack direction="vertical">
+      <VideoPlayer
+        title="GitHub media player"
+        showPlayPauseButton={false}
+        showSeekControl={false}
+        showCCButton={false}
+        showMuteButton={false}
+        showVolumeControl={false}
+        showFullScreenButton={false}
+      >
+        <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+        <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+      </VideoPlayer>
+      <Stack direction="horizontal">
+        <Button onClick={() => togglePlaying()}>{isPlaying ? 'Pause' : 'Play'}</Button>
+        <Button onClick={() => seek(0)}>Go to start</Button>
+        <Button onClick={() => seek(t => t + 5)}>Skip 5 seconds</Button>
+      </Stack>
+    </Stack>
+  )
+}
+
+export const ControlledProgrammatically = () => {
```

Comment:
Just a reminder that you'll need to update snapshots as you're adding a new story,